client/src/components/dashboard/verify/Verify.tsx

```
@@ -5,90 +1,121 @@
+  }, []);
+  // Final verification logic
+  useEffect(() => {
+    if (idStatus === 'Approved' && biodataCreated) {
+      setVerificationStatus('Verified');
+    } else {
+      setVerificationStatus('Not Verified');
+    }
+  }, [idStatus, biodataCreated]);
 
   const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     if (event.target.files && event.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)